components/post_view/post_body/post_body.tsx

```diff
@@ -212,15 +240,12 @@ export default class PostBody extends React.PureComponent {
             <div>
                 {comment}
                 <div
-                    id={`${post.id}_message`}
+                    id={`${post ? post.id : ''}_message`}
```

Comment:
do we want to have `_message` as an id?

Reply:
@Willyfrog - I am not sure about it. I only added null check into it. Please suggest what should we add here.

Reply:
I'd define a constant outside of the div
```js
const divId = post ? `${post.id}_message` : undefined
```
and then the div:
```
<div
    id={divId}
```
that way we don't have the option of repeating IDs

it's either that, or having a different unique identifier that we could rely on when there is no id in the post.